lib/shared/log/toConsole.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+var fancyLog = require('fancy-log');
+
+// The sorting of the levels is
+// significant.
+var levels = [
+  'debug', // Logs all levels.
+  'info',  // Logs info, warn and error.
+  'warn',  // Logs warn and error.
+  'error', // Logs error.
```

Comment:
after reviewing a bunch of the old threads about "a real logger" it seems that @contra wanted errors to be output on stderr.  I think I will add another method on `fancy-log` that calls `console.error` instead of `console.log`